packages/block-editor/README.md

```diff
@@ -415,6 +415,11 @@ _Returns_
 
 -   `Array`: converted rules.
 
+<a name="Typewriter" href="#Typewriter">#</a> **Typewriter**
+
+Ensures that the text selection keeps the same vertical distance from the
```

Comment:
Is this the same vertical distance from the top or bottom edges of the viewport?

Reply:
Both? :)

Reply:
Yes but it feels as if this depends on where my caret is ... In other editors typewriter mode made it always have the current line at 1/2 the viewport size. Here, although I like it, it is varying. What I meant by my comment is to somehow describe this behaviour here with a longer phrase.